docs/framework/database/schema-design.md

```diff
@@ -13,10 +13,10 @@ Another method of viewing CiviCRM's Database model is to use one of various MySQ
 1. Look at the list of tables in the database. The table name generally gives you a good idea of the type of data it stores.
 2. Pick one table and look closer. Let's look at `civicrm_country` (a relatively simple example) by opening the "Table inspector" for that table.
     - We can look at the list of columns in the table.
-    ![MySQL Workbench CiviCRM Country Columns](/img/mysql_workbench_civicrm_country_tables.png)
+    ![MySQL Workbench CiviCRM Country Columns](/docs/img/mysql_workbench_civicrm_country_tables.png)
```

Comment:
Ahhh. Interesting. Ok that makes a lot of sense! Thanks for letting me know about this. I'm going to think on this and see if I can identify a way to prevent confusion like this in the future.